content/source/docs/enterprise-beta/private/install.html.md

```diff
@@ -0,0 +1,153 @@
+standard configuration package uses Terraform to create both the compute and data layer resources, and optionally uses Route53 to configure the DNS automatically.
+
+## Preflight
+
+### Dependencies
+
+Before setup begins, a few resources need to be provisioned. We consider these out of scope for the cluster provisioning because they depend on the user's environment.
+
+The following are required to complete installation:
+
+* **AWS IAM credentials** capable of creating new IAM roles configuring various services. We suggest you use an admin role for this. The credentials are only used for setup; during runtime only an assumed role is used.
+* **AWS VPC** containing at least 2 subnets. These will be used to launch the cluster into. Subnets do not need to be public, but they do need an internet gateway at present.
+* **SSH Key Pair** configured with AWS EC2. This will be used to configure support access to the cluster. This SSH key can be optionally removed from the instance once installation is complete.
+  * To create a new one, [see Amazon's docs for EC2 key pairs.](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html)
+* A **Publicly Trusted TLS certificate** registered with AWS Certificate Manager. This can be one created by ACM for a hostname or the certificate can be imported into it.
+  * To create a new ACM-managed cert: <https://console.aws.amazon.com/acm/home#/wizard/>
+  * To import an existing cert: <https://console.aws.amazon.com/acm/home#/importwizard/>
+  * *NOTE:* Certificates are per region, so be sure to create them in the same region as you'll be deploying Terraform Enterprise into.
+  * *NOTE:* The certificate must allow the fully qualified hostname that Terraform Enterprise will be using. This means you need to decide on a final hostname when creating the certificates and use the same value in the configuration.
+
+The following details will be requested during the application bootstrapping process. It's helpful to have them prepared beforehand.
+
+* **SMTP Credentials**: Terraform Enterprise requires SMTP information to send email. SMTP configuration can be skipped if necessary during the installation, but HashiCorp recommends configuring it during the initial bootstrapping process.
+  * Hostname and port of SMTP service
+  * Type of authentication (plain or login)
+  * Username and password
+  * Email address to use as the sender for emails originating from Terraform Enterprise
+* (Optional) **Twilio Credentials**: Terraform Enterprise can use Twilio for SMS-based 2-factor authentication. If Twilio is not configurated, virtual MFA device-based 2FA (e.g. Google Authenticator) will still be available.
+
+### Required Variables
+
+The following Terraform variables are required inputs and must be populated prior to beginning installation.
+
+The values for these variables should be placed in the `terraform.tfvars` file. Copy `terraform.tfvars.example` to `terraform.tfvars` and edit it with the proper values.
```

Comment:
The biggest thing I'm confused about, as a reader, is where this Terraform working directory actually comes from. Did sales deliver it as a tarball, or is it in a git repo someplace and I need to download it? We could use a note somewhere around here to specify.